src/scancode/utils.py

```diff
@@ -59,13 +59,21 @@ class BaseCommand(click.Command):
     # "Try 'scancode --help' for help on options and arguments."
     short_usage_help = ''
 
+    options = []
```

Comment:
Is there any possible issue with this being a class-level attribute, e.g. a class global?